tests/phpunit/tests/load/wpIsIniValueChangeable.php

```diff
@@ -40,7 +40,7 @@ public function data_wp_is_ini_value_changeable() {
 			array( 'upload_tmp_dir', false ), // PHP_INI_SYSTEM.
 		);
 
-		if ( extension_loaded( 'Tidy' ) && version_compare( PHP_VERSION, '7.0.0', '>' ) ) {
+		if ( extension_loaded( 'Tidy' ) && PHP_VERSION_ID > 70000 ) {
```

Comment:
Is this comparison correct ? Or should it be `>=` ? Or maybe the better question is: why is the PHP 7.0 comparison done in the first place ? The `tidy.clean_output` ini directive was added in PHP 5.0.0 as far as I can see.

Note: I haven't been able to determine whether the "changability" was changed in PHP 7.0. Could be that that's the reason ?

P.S.: 💯 on the principle of this PR!

Reply:
Thanks a lot. I thought to only switch the `phpversion()`/`version_compare` to `PHP_VERSION_ID` and keep it `>` for the parity with existing code.

I don't see anything that changed in the tidy extension between [7.0.0...7.0.1](https://github.com/php/php-src/compare/php-7.0.1...php-7.0.0), and like you said, it's questionable why the PHP 7.0 comparison is done there.

Do you think we should go ahead and change it to `>=`, or perhaps bring it in a separate ticket to remove the version comparison altogether?

Reply:
Might be worth doing a `blame` first to trace back why it was added in the first place and to check that if there was a valid reason, it was added correctly.

Reply:
The oldest commit I could find out was [`c68f1ec772b3`](https://github.com/WordPress/wordpress-develop/commit/c68f1ec772b33f3e5d563936ba4002644c6d5207`), that actually attributes you @jrfnl and @ocean90, but the commit message does not indicate if the `> 7.0.0` was deliberate. The [ticket](https://core.trac.wordpress.org/ticket/32075) links to PHP bug [#44936](https://bugs.php.net/bug.php?id=44936) that reports the ini access value being different in minor versions, so my uninformed gut feeling is to not change the current `>` to `>=`, even though PHP 7.0.0 is too old for us to bother that much.

Reply:
For consistency and micro-optimization, should the conditional check order be flipped?

Reply:
Flipped in 3b1a418.